query/src/servers/http/v1/query/execute_state.rs

```diff
@@ -154,34 +165,38 @@ impl ExecuteState {
                             Ok(block) => tokio::select! {
                                 _ = block_tx.send(block) => { },
                                 _ = abort_rx.recv() => {
-                                    ExecuteState::stop(&state, Err(ErrorCode::AbortedQuery("query aborted")), true).await;
+                                    ExecuteState::stop(&executor, Err(ErrorCode::AbortedQuery("query aborted")), true).await;
                                     break;
                                 },
                             },
                             Err(err) => {
-                                ExecuteState::stop(&state, Err(err), false).await;
+                                ExecuteState::stop(&executor, Err(err), false).await;
                                 break
                             }
                         };
                     } else {
-                        ExecuteState::stop(&state, Ok(()), false).await;
+                        ExecuteState::stop(&executor, Ok(()), false).await;
                         break;
                     }
                 }
                 log::debug!("drop block sender!");
             })?;
-        Ok((state_clone, schema))
+        Ok((executor_clone, schema))
     }
 
-    pub(crate) async fn stop(this: &ExecuteStateRef, reason: Result<()>, kill: bool) {
+    pub(crate) async fn stop(this: &ExecutorRef, reason: Result<()>, kill: bool) {
```

Comment:
this -> self ?

Reply:
type ExecutorRef = Arc<RwLock<Executor>>;
so it is a not method, I saw some suggestion to use `this` in the situation.

Reply:
TIL